CockpitPlugin/src/main/java/com/polidea/cockpitplugin/generator/DebugCockpitGenerator.kt

```diff
@@ -25,58 +26,138 @@ internal class DebugCockpitGenerator : BaseCockpitGenerator() {
                         add(createRemoveSelectionChangeListenerMethodSpecForParam(listTypeParam))
                         add(createSelectedValueGetterMethodSpecForParam(listTypeParam))
                     }
+                    is CockpitColor -> {
+                        shouldGenerateColorFields = true
+                        addGetterSetterAndPropertyChangeListenerMethods(param)
```

Comment:
What about something like this:
```
val colorParam = param as CockpitParam<CockpitColor>
add(createGetterMethodSpecForParam(colorParam.name, colorParam.value.value))
add(createSetterMethodSpecForParam(colorParam.name, colorParam.value.value))
add(createAddPropertyChangeListenerMethodSpecForParam(colorParam.name, colorParam.value.value))
add(createRemovePropertyChangeListenerMethodSpecForParam(colorParam.name, colorParam.value.value))
```
I explained changed method definitions below.